isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.h

```diff
@@ -403,6 +403,12 @@ namespace Isis {
    *                           describing the different file format versions.
    *   @history 2018-01-30 Adam Goins - Ensured point sizes are written/read as lsb by using
    *                           EndianSwapper.
+   *   @history 2018-03-28 Adam Goins - Added targetRadii groups to the header. Changed the
+   *                           versioner to write these values out in a targetRadii group for
+   *                           both binary V0005 and PvlV0005 networks. Fixes #5361.
+   *   @history 2018-04-05 Adam Goins - Added hastargetRadii() and targetRadii() to the versioner
```

Comment:
This feels extremely pedantic, especially after forgetting to review your pull request for over a week, but could you change `hastargetRadii()` in this history comment to `hasTargetRadii()` ?